terraform/modules/helm/secor/secor-helm-chart/values.yaml

```diff
@@ -1,428 +1,410 @@
-cloud_storage_bucket: "${cloud_storage_bucket}"
-namespace: "${secor_namespace}"
+cloud_storage_bucket: "obsrv-dev-725876873105"
+namespace: "secor"
 storageClass: "gp2"
-secor_env: "${deployment_stage}"
-image_repository: "${image_repository}"
-region: "${region}"
+secor_env: "dev"
```

Comment:
Please revert back to using variables from terraform for secor_env, image_repository and secor_env. Please use the "default" as value for storageClass in the values.yaml. It needs to be overridden in each specific cloud provider.